clients/src/main/java/org/apache/kafka/common/requests/DescribeLogDirsResponse.java

```diff
@@ -31,6 +31,7 @@
 public class DescribeLogDirsResponse extends AbstractResponse {
 
     public static final long INVALID_OFFSET_LAG = -1L;
+    public static final long UNKNOWN_VOLUME_BYTES = -1L;
```

Comment:
Do we still want this? 
Correct me if I am wrong here but I thought that we reached a conclusion in the KIP that Optional will cover the scenario when client is using a newer API and broker is old.

Reply:
The wire protocol does not have nullable integers. So the new fields are defined to use -1 as the default value in the protocol: https://github.com/apache/kafka/blob/a73b96a7f6c467b32ca30b93534468fd192eeabe/clients/src/main/resources/common/message/DescribeLogDirsResponse.json#L52-L56

This constant is used to map the value in DescribeLogDirsResponse to an Optional: https://github.com/apache/kafka/blob/a73b96a7f6c467b32ca30b93534468fd192eeabe/clients/src/main/java/org/apache/kafka/clients/admin/LogDirDescription.java#L44-L45